packages/react-scripts/scripts/build-module.js

```diff
@@ -1,58 +1,82 @@
 require('../utils/loadEnv');
 
-var paths = require('../config/paths');
+var path = require('path');
+var fs = require('fs-extra');
+var chokidar = require('chokidar');
+var transformFileSync = require('babel-core').transformFileSync;
 var spawn = require('cross-spawn');
-var result;
+var paths = require('../config/paths');
 
-var defaults = [
-  '--presets=trunkclub',
-  '--source-maps'
-];
+function lint () {
+  return spawn.sync('node', [require.resolve('./lint')], {stdio: 'inherit'});
+}
 
-var directoryArgs = [
-  paths.appSrc,
-  '--out-dir=' + paths.appBuild
-];
+function transformWithBabel (filePath) {
+  const { code } = transformFileSync(path.join(paths.appSrc, filePath), {
+    ast: false,
+    sourceMaps: 'inline',
```

Comment:
I think it's just a loading thing. if the sourcemaps are separate I am guessing they wont' be loaded unless you open devtools? babel outputs sourcemaps separately in the cli, guessing it's a similar option in the node api.